db/structure.sql

```diff
@@ -47,7 +47,7 @@ CREATE TYPE public.lang AS ENUM (
     'android',
     'others',
     'unassigned',
-    'react_native'
+    'vuejs'
```

Comment:
Why this change?

Reply:
It seemed like it was deleted in the previous PR, dont know why. @santib commented on that on the previous PR. And the order changed probably is because postgres version or something like that

Reply:
i will check, it happened the same on the other PR